packages/amplify-graphql-api-construct-tests/src/__tests__/owner-auth/assoc-field/API.ts

```diff
@@ -0,0 +1,1090 @@
+/* tslint:disable */
```

Comment:
Note that I've edited the queries to removed createdAt, updatedAt, and __typename fields to clean up the interface and remove the need to model those in SQL DBs.